shared/shared_defs.h

```diff
@@ -28,8 +28,7 @@ typedef enum {
     STATUS_IDLE,
     STATUS_HOMING,
     STATUS_MOVING,
-    STATUS_MEASURING,
-    STATUS_LIMIT_REACHED
+    STATUS_FAULT
```

Comment:
Do we no longer need to indicate status for data acquisition (ie. `STATUS_MEASURING`)?

Reply:
At least for now, the Arduino has no concept of data acquisition so I removed it